### DISCUSSION TITLE
Очень прошу помочь!

### DISCUSSION THREAD

**Discussion by fizsef** [AdguardTeam/AdGuardHome#5511] [Q&A]
Установил AGH на роутер по инструкции от OpenWrt  https://openwrt.org/docs/guide-user/services/dns/adguard-home
OpenWrt 22.03.3 .
Но есть вопросы, которые не могу решить. 
1. Как избавиться от таких ошибок  - 
2. `Sun Feb 19 17:43:34 2023 daemon.err AdGuardHome[4182]: 2023/02/19 14:43:34.182927 [info] Listening to udp://[::]:53
Sun Feb 19 17:43:34 2023 daemon.err AdGuardHome[4182]: 2023/02/19 14:43:34.183050 [info] Creating a TCP server socket
Sun Feb 19 17:43:34 2023 daemon.err AdGuardHome[4182]: 2023/02/19 14:43:34.183366 [info] Listening to tcp://[::]:53
Sun Feb 19 17:43:34 2023 daemon.err AdGuardHome[4182]: 2023/02/19 14:43:34.186286 [info] Entering the UDP listener loop on [::]:53
Sun Feb 19 17:43:34 2023 daemon.err AdGuardHome[4182]: 2023/02/19 14:43:34.187133 [info] Entering the tcp listener loop on [::]:53`   

3. Как настроить, чтобы в стастистрику и журнал не попадали такие запросы - 
![1](https://user-images.githubusercontent.com/74646605/219955581-14f2d67a-4b02-44e8-88b2-213ffecd40c9.jpg)

**Comment by ainar-g** [comment]
1.  Не совсем понимаю, что вы имеете в виду под ошибками.  Это информационный вывод.

2.  Используйте поле `ignored` в настройках статистики и лога.  Они были добавлены в релизе v0.107.24.

**Comment by fizsef** [comment]
Спапсибо большое. 
Только не понял из описания в вики по конфиг файлу, по опции ignored . Будут игнорированы запросы от указанных в этом поле айпи(хостов) , или запросы к указанным хостам. Т.е. нужно указывать источнки запросов, или целевые хосты? 
Но из-за этой ошибки при проверке обновлени я не могу повысить версию и  вернулся на 107.21 .
Как исправить эту ошибку (когда будет исправлена)?
![1](https://user-images.githubusercontent.com/74646605/222237463-217ea5b7-726e-454a-b413-3c9971134116.jpg)

**Reply by ainar-g** [reply]
(Лучше отвечать в тред, а не писать новые сообщения.)

Указывать нужно данные запроса.  То есть, если вы не хотите видеть запросы от любых клиентов к `www.google.com`, то его надо добавить в `querylog.ignored`.

Сообщение говорит о том, что ваш AGH не может достучаться до нашего сервера обновлений.  Если сообщение не проходит, то возможно его адрес заблокирован вашим провайдером.  Если это так, можете либо использовать прокси, либо отключить автообновление и воспользоваться [ручным обновлением][1].

[1]: https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#manual-update
